eth/backend.go

```diff
@@ -76,9 +79,9 @@ type Ethereum struct {
 	snapDialCandidates enode.Iterator
 	merger             *consensus.Merger
 
+	historicalRPCService *rpc.Client
```

Comment:
I think not changing the order is easy to compare with OP when doing upstream works